node-operators/block-producers/operational-tasks/upgrade-your-node.md

```diff
@@ -0,0 +1,156 @@
+---
+title: Upgrade your Node
+description: Follow these instructions to update your node to the latest version of the Tanssi client software you use to produce blocks on the Tanssi Appchain protocol. 
+---
+
+# Upgrade your Node
+
+## Introduction {: #introduction }
+
+This tutorial covers upgrading your Tanssi block producer node. This guide assumes that you've already set up your account and spun up a [block producer node](/node-operators/block-producers/onboarding/run-a-block-producer){target=\_blank}. This guide will show you how to upgrade your node whether you're using [Docker](/node-operators/block-producers/onboarding/run-a-block-producer/block-producer-docker/){target=\_blank} or [Systemd](/node-operators/block-producers/onboarding/run-a-block-producer/block-producer-systemd/){target=\_blank}. 
```

Comment:
Maybe add a bit on why you may need to upgrade your node, or information on how block producers will be notified if a upgrade is required (on discord, I assume?)

I think this intro feels a bit choppy since each sentence starts with "This guide" or "This tutorial". I think it could benefit from being rewritten a bit so it flows a bit better.